src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -167,7 +162,23 @@ export class MeetingManager implements AudioVideoObserver{
   async initializeMeetingSession(
     configuration: MeetingSessionConfiguration
   ): Promise<any> {
-    const logger = new ConsoleLogger('SDK', this.logLevel);
+    const consoleLogger = new ConsoleLogger('SDK', this.logLevel);
```

Comment:
It would be nice if we abstract out the logger's into a separate function which give us the postLogger and consoleLogger instances. Not necessary but will keep it separate I guess.

Reply:
Good idea, I've separated it into it's own method so the initialize function stays a bit cleaner